src/item.rs

```diff
@@ -1092,6 +1092,17 @@ pub mod parsing {
             let content;
             let paren_token = parenthesized!(content in input);
             let inputs = content.parse_terminated(FnArg::parse)?;
+            let variadic: Option<Token![...]> = match inputs.last() {
+                Some(punctuated::Pair::End(&FnArg::Captured(ArgCaptured {
+                    ty: Type::Verbatim(TypeVerbatim { ref tts }), ..
+                }))) => {
+                    match parse2::<token::Dot3>(tts.clone()) {
+                        Ok(tok) => Some(tok),
```

Comment:
This could be written more compactly as `parse2(tts.clone()).ok()`.

Reply:
:+1: good point